pkgs/tools/inputmethods/libkkc/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl
+, vala, gobjectIntrospection, intltool, python2Packages, glib
+, pkgconfig
+, gnome3, json_glib, marisa, libkkc-data
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libkkc";
+  version = "0.3.5";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "${meta.homepage}/releases/download/v${version}/${name}.tar.gz";
+    sha256 = "89b07b042dae5726d306aaa1296d1695cb75c4516f4b4879bc3781fe52f62aef";
+  };
+
+  nativeBuildInputs = [
+    vala gobjectIntrospection
+    python2Packages.python python2Packages.marisa
+    intltool glib pkgconfig
+  ];
+
+  buildInputs = [ marisa libkkc-data ];
+
+  propagatedBuildInputs = [ gnome3.libgee json_glib ];
```

Comment:
Just use `libgee` and `inherit (gnome3) libgee;` in `all-packages.nix` instead; we try to avoid using whole (sub-)package sets as arguments.

Reply:
fixed